tests/lang_callable/macros/tif_ill_formed.nim

```diff
@@ -0,0 +1,17 @@
+discard """
+  description: '''
+    Ensure that ill-formed if-statement AST created by macros is detected
+  '''
+  cmd: "nim check --msgFormat:sexp --filenames=canonical --hints:off $options $file"
+  nimoutformat: sexp
```

Comment:
```suggestion
  cmd: "nim check --hints:off $options $file"
  nimoutfull: "true"
```
The `sexp` mode and related testament support is a bit shaky at the moment, the general direction is to not use it for now.